app/views/standard/addClosureMetadata.scala.html

```diff
@@ -11,94 +11,99 @@
-                                    "number",
-                                    fieldErrors,
-                                    Map(Symbol("_dynamicForm") -> true),
-                                    inputMode,
-                                    addSuffixText = true
-                                )
-                                case DropdownField(fieldId, fieldName, fieldDescription, options, selectedOption, _, fieldErrors) =>
-                                    //could theoretically be a checkbox or radio (since radios are not always boolean)
-                                    inputDropdown(
-                                        fieldId,
-                                        fieldName,
-                                        fieldDescription,
-                                        options,
-                                        selectedOption,
-                                        Map(
-                                            Symbol("_disabledOption") -> "",
-                                            Symbol("_dynamicForm") -> true
-                                        ),
-                                        fieldErrors
-                                    )
-                            }
-                        }
-                    }
-                    <div class="govuk-button-group">
-                            <!-- Continue -->
-                        <button data-prevent-double-click="true" class="govuk-button" type="submit" data-module="govuk-button" role="button">
-                            Continue
-                        </button>
-                            <!-- Redirects to Homepage for now but will probably just take you back to previous page -->
-                        <a class="govuk-link" href="@routes.HomepageController.homepage()">Cancel</a>
-                    </div>
-                </div>
+                case DateField(fieldId, fieldName, fieldDescription, day, month, year, _, fieldErrors, _) => inputDate(
+                  fieldId,
+                  fieldName,
+                  fieldDescription,
+                  day,
+                  month,
+                  year,
+                  fieldErrors,
+                  Map(Symbol("_dynamicForm") -> true)
+                )
+                case TextField(fieldId, fieldName, fieldDescription, inputNameAndValue, inputMode, _, fieldErrors) => inputText(
+                  fieldId,
+                  fieldName,
+                  fieldDescription,
+                  inputNameAndValue,
+                  "number",
+                  fieldErrors,
+                  Map(Symbol("_dynamicForm") -> true),
+                  inputMode,
+                  addSuffixText = true
+                )
+                case DropdownField(fieldId, fieldName, fieldDescription, options, selectedOption, _, fieldErrors) =>
+                  //could theoretically be a checkbox or radio (since radios are not always boolean)
```

Comment:
Is this comment necessary?

Reply:
I added this originally before Vimlesh refactored it, so it made more sense as my version was  looser in how it constructed fields (was a bit more data-driven); I made the assumption that `"propertyType": "Defined",` &. `"dataType": "Text",` = "dropdown" but I was letting the user know that this doesn't have to always be the case, and that it could be changed. The comment probably makes more sense [here](https://github.com/nationalarchives/tdr-transfer-frontend/blob/3e18b4e595b268560b744f25a1680b102becbaa0/app/controllers/util/CustomMetadataUtils.scala#L52)

Reply:
I don't think so, I've deleted it.